visualizer-server/src/metrics.rs

```diff
@@ -0,0 +1,42 @@
+use actix_web::{dev::Server, App, HttpServer};
+use actix_web_prom::{PrometheusMetrics, PrometheusMetricsBuilder};
+use std::net::SocketAddr;
+
+#[derive(Clone)]
+pub struct Metrics {
+    metrics_middleware: PrometheusMetrics,
```

Comment:
I don't understand, why two?

Reply:
oh is that something like we need separate thing to pass into the server?

Reply:
the first one will host metrics on 0.0.0.0:6000
the second one will add middleware to HTTP server and count default metrics like requests count and duration